operators/pm.go

```diff
@@ -39,25 +41,26 @@ func (o *pm) Evaluate(tx *coraza.Transaction, value string) bool {
 	return pmEvaluate(o.matcher, tx, value)
 }
 
-var _ coraza.RuleOperator = (*pm)(nil)
-
 func pmEvaluate(matcher ahocorasick.AhoCorasick, tx *coraza.Transaction, value string) bool {
 	var numMatches int
```

Comment:
Just nit but since it's not used in the non-capturing case let's move below

Reply:
Done.